tables.py

```diff
@@ -0,0 +1,16 @@
+import database
+
+def create_guild_tables(guild_id):
```

Comment:
Could this just go in database.py?

Reply:
It could, database.py is going to be pretty full of stuff once more tables are added. I was thinking of ways to organize this, I added that debug line in database.py just to check if that was getting run more than once to see if it needed a refactoring.

Was thinking each table connection could be a singleton class, might make cogs a little messy if they use more than one table though.

Reply:
That works. Could you maybe rename/restructure database.py so I can see what kind of structure to use?

Reply:
Made a stab at a singleton connection, and separate classes for each table.